electrum_nmc/electrum/servers.json

```diff
@@ -104,20 +104,20 @@
         "s": "57002",
         "version": "1.4"
     },
-    "t3ts346cpsdbqjkcvyveqfgldtipoco5yzxqs6sted6y4fgsgasnp3ad.onion": {
+    "wirg2tsto7rme7n26lkd3ivbvxmjyy2pktlozwjuep22jcsfsghfqbqd.onion": {
         "pruning": "-",
         "s": "50002",
-        "version": "1.4"
+        "version": "1.4.3"
     },
     "ulrichard.ch": {
         "pruning": "-",
         "s": "50006",
         "t": "50005",
-        "version": "1.4"
+        "version": "1.4.3"
```

Comment:
Is it intentional that you're updating ulrichard's version? If so, can you split that one into a separate PR?